client/apps/edit/components/section_container/index.coffee

```diff
@@ -61,6 +63,7 @@ module.exports = React.createClass
           when 'fullscreen' then SectionFullscreen
           when 'callout' then SectionCallout
           when 'toc' then SectionToc
+          when 'image' then SectionImage
```

Comment:
Ah see line 18 and 69, I've added these already from [this PR](https://github.com/artsy/positron/pull/917/commits/917cf32ef04dafd4313cd445b5092a4643ed81a1).